docs/general/how-to-define-semantic-conventions.md

```diff
@@ -0,0 +1,151 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: How To Define New Semantic Conventions
+aliases: [how-to-define-new-semantic-conventions]
+--->
+
+# How to define new semantic conventions
+
+**Status**: [Development][DocumentStatus]
+
+<!-- toc -->
+
+- [Defining new conventions](#defining-new-conventions)
+  - [Best practices](#best-practices)
+    - [Defining attributes](#defining-attributes)
+    - [Defining spans](#defining-spans)
+    - [Defining metrics](#defining-metrics)
+    - [Defining resources](#defining-resources)
+    - [Defining events](#defining-events)
+- [Stabilizing existing conventions](#stabilizing-existing-conventions)
+  - [Migration plan](#migration-plan)
+
+<!-- tocstop -->
+
+This document describes requirements, recommendations, and best practices on how to define conventions
+for new areas or make substantial changes to the existing ones.
+
+## Defining new conventions
+
+- New conventions MUST have a group of codeowners. See [project management](https://github.com/open-telemetry/community/blob/main/project-management.md) for more details.
+  <!-- TODO: add CI check for CODEOWNERS file (when a new area is added) -->
+- New conventions SHOULD be defined in YAML files. See [YAML Model for Semantic Conventions](/model/README.md) for the details.
+- New conventions SHOULD be defined with `development` stability level.
+- New conventions SHOULD include telemetry signal definitions (spans, metrics, events, resources, profiles) and MAY include new attribute definitions.
+
+### Best practices
+
+#### Defining attributes
+
+Reuse existing attributes when possible. Look through [existing conventions](/docs/attributes-registry/) for similar areas,
+check out [common attributes](/docs/general/attributes.md).
+Semantic conventions authors are encouraged to use attributes from different namespaces.
+
+Consider adding a new attribute when the following apply:
+
+- It provides a clear benefit to end users by enhancing telemetry.
+- There is a clear plan to use the attributes when defining spans, metrics, events, resources, or other telemetry signals in semantic conventions.
+- There is a clear plan on how these attributes will be used by instrumentations
```

Comment:
are these criteria connected with `AND`, should they all apply? I would recommend to specify it explicitly to avoid confusion